lib/logstorage/syslog_parser.go

```diff
@@ -281,7 +293,9 @@ func (p *SyslogParser) parseRFC5424SDLine(s string) (string, bool) {
 		return s, false
 	}
 
-	sdValue := strings.TrimSpace(s[:i])
+	fmt.Println(strings.TrimSpace(s[:i]))
```

Comment:
I assume it's ready for review. Please remove debug codes L296 & L298

Reply:
done